gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/GobblinOutputCommitter.java

```diff
@@ -61,10 +62,10 @@ public void abortJob(JobContext jobContext, JobStatus.State state) throws IOExce
     FileSystem fs = FileSystem.get(fsUri, conf);
 
     Path mrJobDir = new Path(conf.get(ConfigurationKeys.MR_JOB_ROOT_DIR_KEY), conf.get(ConfigurationKeys.JOB_NAME_KEY));
-    Path jobInputDir = new Path(mrJobDir, "input");
+    Path jobInputDir = new Path(mrJobDir, MRJobLauncher.INPUT_DIR_NAME);
 
-    if (!fs.exists(jobInputDir) || !fs.getFileStatus(jobInputDir).isDir()) {
-      LOG.warn("Folder " + jobInputDir + " containing serialized WorkUnits doesn't exist. No data will cleaned up.");
+    if (!fs.exists(jobInputDir) || !fs.isDirectory(jobInputDir)) {
```

Comment:
`fs.isDirectory` is a simpler way of doing the same thing and `isdir` in the original code has been deprecated.